### DISCUSSION TITLE
How do I Update Z-Wave JS?

### DISCUSSION THREAD

**Discussion by stockhausenj** [zwave-js/zwave-js-ui#4049] [General]
Hey!

I'm running Z-Wave JS UI using the `latest` version of the container image.

Is there a way to update Z-Wave JS UI to use the latest release of Z-Wave JS? I'm trying to go from 14.3.6 to 14.3.7. Or do I have to wait until a new image of Z-Wave JS UI is released to Docker Hub?

**Comment by robertsLando** [comment]
> Or do I have to wait until a new image of Z-Wave JS UI is released to Docker Hub?

You have to wait, usually I release a new version the same day the driver ships a new one :)

**Reply by robertsLando** [reply]
@stockhausenj FYI I release a new version yesterday with latest driver :)
